examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,235 @@
+🐳  Preparing Kubernetes v1.15.4 on Docker 18.09.9 ...
+🚜  Pulling images ...
+🚀  Launching Kubernetes ...
+⌛  Waiting for: apiserver proxy etcd scheduler controller dns
+🏄  Done! kubectl is now configured to use "minikube"
+```
+
+2. Download the latest version of Kuma
+
+```
+$ wget https://kong.bintray.com/kuma/kuma-0.2.2-darwin-amd64.tar.gz
+--2019-10-13 05:53:46--  https://kong.bintray.com/kuma/kuma-0.2.2-darwin-amd64.tar.gz
+Resolving kong.bintray.com (kong.bintray.com)... 52.88.33.18, 54.200.232.13
+Connecting to kong.bintray.com (kong.bintray.com)|52.88.33.18|:443... connected.
+HTTP request sent, awaiting response... 302
+Location: https://akamai.bintray.com/69/694567d6d0d64f5eb5a5841aea3b4c3d60c8f2a6e6c3ff79cd5d580edf22e12b?__gda__=exp=1570917947~hmac=68f26ab23b95f97acebfc4b33a1bc1e88aeca46a44b1bc349af851019c941d0a&response-content-disposition=attachment%3Bfilename%3D%22kuma-0.2.2-darwin-amd64.tar.gz%22&response-content-type=application%2Fgzip&requestInfo=U2FsdGVkX1_SREBFG76q54ykX416x4BKSbGVrX5A-GfV55I-FdyX_0L9WI3EaLJdsXfRQ4V2pY3vP9viaRvtUxQEjLKVz_AEytCDaz5VW3oTvdhio0sq10KPgW3Z3hFN&response-X-Checksum-Sha1=01c56caae58a6d14a1ad24545ee0b25421c6d48e&response-X-Checksum-Sha2=694567d6d0d64f5eb5a5841aea3b4c3d60c8f2a6e6c3ff79cd5d580edf22e12b [following]
+--2019-10-13 05:53:47--  https://akamai.bintray.com/69/694567d6d0d64f5eb5a5841aea3b4c3d60c8f2a6e6c3ff79cd5d580edf22e12b?__gda__=exp=1570917947~hmac=68f26ab23b95f97acebfc4b33a1bc1e88aeca46a44b1bc349af851019c941d0a&response-content-disposition=attachment%3Bfilename%3D%22kuma-0.2.2-darwin-amd64.tar.gz%22&response-content-type=application%2Fgzip&requestInfo=U2FsdGVkX1_SREBFG76q54ykX416x4BKSbGVrX5A-GfV55I-FdyX_0L9WI3EaLJdsXfRQ4V2pY3vP9viaRvtUxQEjLKVz_AEytCDaz5VW3oTvdhio0sq10KPgW3Z3hFN&response-X-Checksum-Sha1=01c56caae58a6d14a1ad24545ee0b25421c6d48e&response-X-Checksum-Sha2=694567d6d0d64f5eb5a5841aea3b4c3d60c8f2a6e6c3ff79cd5d580edf22e12b
+Resolving akamai.bintray.com (akamai.bintray.com)... 104.93.1.149
+Connecting to akamai.bintray.com (akamai.bintray.com)|104.93.1.149|:443... connected.
+HTTP request sent, awaiting response... 200 OK
+Length: 42892462 (41M) [application/gzip]
+Saving to: ‘kuma-0.2.2-darwin-amd64.tar.gz’
+
+kuma-0.2.2-darwin-amd64.tar.g 100%[===============================================>]  40.91M  2.61MB/s    in 20s
+
+2019-10-13 05:54:08 (2.09 MB/s) - ‘kuma-0.2.2-darwin-amd64.tar.gz’ saved [42892462/42892462]
+```
+
+3. Unbundle the files to get the following components:
+
+```
+$ tar xvzf kuma-0.2.2-darwin-amd64.tar.gz
+x ./
+x ./conf/
+x ./conf/kuma-cp.conf
+x ./bin/
+x ./bin/kuma-dp
+x ./bin/envoy
+x ./bin/kuma-tcp-echo
+x ./bin/kumactl
+x ./bin/kuma-cp
+x ./README
+x ./LICENSE
+```
+
+4. Go into the /bin directory where the kuma components will be:
+
+```
+$ cd bin && ls
+envoy   kuma-cp   kuma-dp   kuma-tcp-echo kumactl
+```
+
+5. Install the control plane using `kumactl`
+
+```
+$ kumactl install control-plane | kubectl apply -f -
```

Comment:
This command will not work out-of-the box (`kumactl` is not on the `PATH`)

So, either give instructions how to add `bin` folder to the `PATH` or use `./kumactl`